src/main/java/plugily/projects/buildbattle/commands/arguments/admin/plot/AddPlotArgument.java

```diff
@@ -56,15 +55,17 @@ public void execute(CommandSender sender, String[] args) {
           new MessageBuilder("COMMANDS_NO_ARENA_LIKE_THAT").asKey().send(sender);
           return;
         }
+        Player player = (Player) sender;
         CuboidSelector.Selection selection = registry.getPlugin().getCuboidSelector().getSelection(player);
         if(selection == null || selection.getFirstPos() == null || selection.getSecondPos() == null) {
           new MessageBuilder("&cPlease select both corners before adding a plot!").send(sender);
           return;
         }
         FileConfiguration config = ConfigUtils.getConfig(registry.getPlugin(), "arenas");
         int id = 0;
-        if(config.isConfigurationSection("instances." + arena.getId() + ".plots")) {
-          id = config.getConfigurationSection("instances." + arena.getId() + ".plots").getKeys(false).size() + 1;
+        org.bukkit.configuration.ConfigurationSection section = config.getConfigurationSection("instances." + arena.getId() + ".plots");
```

Comment:
why not using ConfigurationSection as import

Reply:
Same as above

Reply:
About import, I would like to see it imported as it looks better and cleaner